packages/mdc-textfield/outline/mdc-text-field-outline.scss

```diff
@@ -33,6 +33,10 @@
 }
 
 .mdc-text-field__outline {
+  @include mdc-ripple-surface;
+  @include mdc-states-base-color(black);
```

Comment:
Remove this line; the next line will already result in including `mdc-states-base-color(text-primary-on-light)`.

Reply:
True (I hadn't tested the behavior prior to writing that comment), but should it be using `text-primary-on-light` like you were doing with `mdc-states`, or `black`?